apps/modernization-ui/src/apps/search/laboratory-report/labReportFormTypes.ts

```diff
@@ -23,9 +24,11 @@ type General = {
     enteredBy?: Selectable[];
     entryMethods?: Selectable[];
     eventDate?: EventDate;
+    eventId?: LabReportEventId | null;
```

Comment:
issue: Let's try to reduce the usage of the generated GraphQL types as much as possible.  The `identification` property is the same as `eventId`, the transformer already converts to it